resources/assets/js/store/modules/web-socket.js

```diff
@@ -68,14 +69,28 @@ export default {
 
     async privateChat({commit}, data) {
       await WS.connect();
-     let subscribe = await WS.subscribe('chat');
+     let subscribe = await WS.subscribe(`chat:private:${data.receiver_id}:${data.sender_id}`);
+     commit('SOCKET_SET_SENDING_MESSAGE', data);
      subscribe.emit("chatMessage", data);
     },
 
     async getPrivateChat({commit}, data) {
       await WS.connect();
-      let subscribe = await WS.subscribe('chat');
+      let subscribe = await WS.subscribe(`chat:private:${data.receiver_id}:${data.sender_id}`);
+      commit('SOCKET_SET_RECEIVING_MESSAGE', data);
```

Comment:
Why are you committing to vue since on refresh will pull our everything 
if you are looking towards mutating messages when sent you can store them inside indexDB then send to server

Reply:
On pull to refresh, u still need to change state so update can be tracked.